aten/src/ATen/native/cpu/PowKernel.cpp

```diff
@@ -2,7 +2,7 @@
 #include <ATen/Dispatch.h>
 #include <ATen/Parallel.h>
 #include <ATen/cpu/vec256/vec256.h>
-#include <ATen/native/TensorIterator.h>
+#include <ATen/native/TensorIterator.h> 
```

Comment:
Trailing whitespace seems to have crept in. Maybe the style checks don't check for trailing whitespace after `#include`s

Reply:
> ##  CI failures summary and remediations
> As of commit [530278a](https://github.com/pytorch/pytorch/commit/530278a047a02f6b31c8431643f7119654cf9c83) (more details [on the Dr. CI page](https://our.intern.facebook.com/intern/opensource/dr_ci/commit/167825833786582/530278a047a02f6b31c8431643f7119654cf9c83/)):
> 
> * **1/1** failures introduced in this PR
> 
> ### 1 failure _not_ recognized by patterns:
> Job	Step	Action
> ![CircleCI](https://avatars0.githubusercontent.com/ml/7?s=12) pytorch_linux_xenial_py3_clang5_asan_test2	Run tests	[repeat rerun](https://our.intern.facebook.com/intern/opensource/ci/restart/160979675801588/)
> This comment was automatically generated by [Dr. CI](https://code.facebook.com/ci/dr-ci-info/) (expand for details).

The failure cause for  `pytorch_linux_xenial_py3_clang5_asan_test2` is build failures of `arm64` builds for `iOS`. 
However, on CircleCI, they've been failing for other PRs as well, so this failure is unrelated to the changes.